x/ibc-transfer/keeper/keeper.go

```diff
@@ -80,7 +80,7 @@ func (k Keeper) ReceiveExecuted(ctx sdk.Context, packet ibcexported.PacketI, ack
 	if !ok {
 		return sdkerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, "channel capability could not be retrieved for packet")
 	}
-	return k.channelKeeper.ReceiveExecuted(ctx, chanCap, packet, acknowledgement)
+	return k.channelKeeper.ReceiveExecuted(ctx, chanCap, packet)
```

Comment:
this wrapper function is not being used on ibc-transfer. cc @cwgoes @colin-axner. Is this problematic?

Reply:
It should be deleted I think. It should be replaced with `WriteAcknowledgement` which for ibc-transfer should be called in `OnRecvPacket` if I am not mistaken.

Reply:
Aye, certainly if it's not used we should eliminate it, and it's not in the spec, `RecvPacket` then `WriteAcknowledgement` is better, although those could be contained in a single wrapper function for convenience / abstraction of synchronous replies